pkg/clusteragent/orchestrator/controller.go

```diff
@@ -134,13 +151,41 @@ func (o *Controller) processPods() {
 		return
 	}
 
-	extraHeaders := map[string]string{
-		api.HostHeader:           o.hostName,
-		api.ContainerCountHeader: "0",
-		api.ClusterIDHeader:      o.clusterID,
-	}
 	for _, m := range msg {
-		// TODO: handle failure & retries
-		o.apiClient.PostMessage(o.processConfig.OrchestratorEndpoints, "/api/v1/orchestrator", m, extraHeaders)
+		extraHeaders := make(http.Header)
+		extraHeaders.Set(api.HostHeader, o.hostName)
+		extraHeaders.Set(api.ClusterIDHeader, o.clusterID)
+
+		body, err := encodePayload(m)
+		if err != nil {
+			log.Errorf("Unable to encode message: %s", err)
+			continue
+		}
+
+		payloads := forwarder.Payloads{&body}
+		responses, err := o.forwarder.SubmitPodChecks(payloads, extraHeaders)
```

Comment:
```suggestion
		_, err := o.forwarder.SubmitPodChecks(payloads, extraHeaders)
```
we can ignore responses here and remove the range below

Reply:
I had left that in as a precautionary measure in case the resulting channel was changed to be unbuffered or of a different size than it currently is.  As of now, the channel is sized to the number of output transactions so none of the worker threads will be blocked writing to it but if the channel size was changed, the worker threads could be blocked if nothing is consuming from the channel.

Thoughts?

Reply:
sounds good to me, let's keep it 👍